Lesson22/todo-app/Gokhan/script.js

```diff
@@ -0,0 +1,67 @@
+// const sayHelloBtn = document.getElementById('btnSayHello');
+
+// sayHelloBtn.addEventListener('cli('ck', function (e) {
+//   console.log('Hello', e);
+// });
+
+// const sayByeBtn = document.getElementById('btnSayBye');
+
+// sayByeBtn.addEventListener('click', function () {
+//   console.log('Bye bye');
+// });
+
+const form = document.getElementById('toDoForm');
+
+form.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event) {
+
+	// Usually you always need it for form submit events!
+
+	event.preventDefault();
+
+	const input = document.getElementById('toDoInput');
+
+	const listItem = document.createElement('li');
+	listItem.innerText = input.value;
```

Comment:
// To fix the issue with the button being crossed, add a span element for the text of the todo:
```suggestion
	const spanElement = document.createElement('span');
	spanElement.innerText = input.value;
	
	listItem.append(spanElement);
```